src/sampling.jl

```diff
@@ -504,9 +508,11 @@ naive_wsample_norep!(a::AbstractArray, wv::WeightVec, x::AbstractArray) =
 #     URL http://www.sciencedirect.com/science/article/pii/S002001900500298X
 #
 # Instead of keys u^(1/w) where u = random(0,1) keys w/v where v = randexp(1) are used.
-function efraimidis_a_wsample_norep!(rng::AbstractRNG, a::AbstractArray, wv::WeightVec, x::AbstractArray)
+function efraimidis_a_wsample_norep!(rng::AbstractRNG, a::AbstractArray,
+                                     wv::WeightVec, x::AbstractArray)
     n = length(a)
-    length(wv) == n || throw(DimensionMismatch("a and wv must be of same length (got $n and $(length(wv)))."))
+    length(wv) == n || throw(DimensionMismatch(
+	    "a and wv must be of same length (got $n and $(length(wv)))."))
```

Comment:
This line is unrelated, please leave it as-is. (Also, in this case, the break should be done after the `||`, or the long `if.. end` form should be used.)